### ISSUE TITLE
EAS Build for iOS throwing error for `paid Apple Developer account` even if i have paid account

### ISSUE THREAD

**Issue by tushar21** [expo/eas-cli#1776]
### Build/Submit details page URL

https://expo.dev/accounts/aperadev/projects/tracker/builds/843de23c-3a91-4830-b5cf-9ba1c64e6ad9

### Summary

I am trying to build for iOS using command `eas build -p ios` and I have paid developer account. But still eas build throwing below error: 
![image](https://user-images.githubusercontent.com/19849521/230562789-6df4cf71-e02e-4a6e-ba77-810817c06381.png)

### Managed or bare?

Managed

### Environment

There is a new version of expo-cli available (6.3.2).                 │
│   You are currently using expo-cli 6.2.1                                │
│   Install expo-cli globally using the package manager of your choice;   │
│   for example: `npm install -g expo-cli` to get the latest version 

### Error output

Authentication with Apple Developer Portal failed!
You have no team associated with your Apple account, cannot proceed.
(Do you have a paid Apple Developer account?)

### Reproducible demo or steps to reproduce from a blank project

Just run command : `eas build -p ios`

**Comment by dsokal** [comment]
Hey, 
- What version of `eas-cli` do you have?
- Can you please share the output of `npx expo-env-info`? It should look sth like this:

```
  expo-env-info 1.0.5 environment info:
    System:
      OS: macOS 13.2.1
      Shell: 5.8.1 - /bin/zsh
    Binaries:
      Node: 16.13.2 - ~/.volta/tools/image/node/16.13.2/bin/node
      Yarn: 1.22.17 - ~/.volta/tools/image/yarn/1.22.17/bin/yarn
      npm: 8.1.2 - ~/.volta/tools/image/node/16.13.2/bin/npm
    Managers:
      CocoaPods: 1.11.3 - /opt/homebrew/lib/ruby/gems/2.7.0/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 22.4, iOS 16.4, macOS 13.3, tvOS 16.4, watchOS 9.4
      Android SDK:
        API Levels: 30, 31, 32
        Build Tools: 29.0.2, 30.0.3, 32.0.0, 32.1.0
        System Images: android-32 | Google APIs ARM 64 v8a
        Android NDK: 24.0.8215888
    IDEs:
      Android Studio: 2021.1 AI-211.7628.21.2111.8309675
      Xcode: 14.3/14E222b - /usr/bin/xcodebuild
    npmPackages:
      expo: ~47.0.12 => 47.0.13 
      react: 18.1.0 => 18.1.0 
      react-native: 0.70.5 => 0.70.5 
    Expo Workflow: managed
```

**Comment by tushar21** [comment]
- eas-cli/3.9.0
- ```
expo-env-info 1.0.5 environment info:
    System:
      OS: Linux 5.4 Ubuntu 20.04.6 LTS (Focal Fossa)
      Shell: 5.0.17 - /bin/bash
    Binaries:
      Node: 18.8.0 - /usr/local/bin/node
      Yarn: 1.22.19 - /usr/bin/yarn
      npm: 9.3.1 - /usr/local/bin/npm
    npmPackages:
      @expo/webpack-config: ^18.0.1 => 0.17.4 
      expo: ^48.0.0 => 47.0.13 
      react: 18.2.0 => 18.1.0 
      react-dom: 18.2.0 => 18.1.0 
      react-native: 0.71.6 => 0.70.5 
      react-native-web: ~0.18.11 => 0.18.12 
    npmGlobalPackages:
      eas-cli: 3.9.0
      expo-cli: 6.2.1
    Expo Workflow: managed

```

**Comment by tushar21** [comment]
Fixed. Issue was my developer account was approved by apple team. After contacting support team, i was able to build.

**Comment by wuchangming** [comment]
> Fixed. Issue was my developer account was approved by apple team. After contacting support team, i was able to build.

Could you please explain how to solve it in detail

**Comment by marianapmonteiro** [comment]
I'm having the same problem. I'm on a team with the role of Admin, but I'm getting this error. EAS CLI version: 5.9.1.

**Comment by MatkoMilic** [comment]
Same issue here..

**Comment by andrew-hossack** [comment]
I love issues where people solve it and provide no more context.

**Comment by MatkoMilic** [comment]
I solved it after a lot of struggles for a week and angry client. @andrew-hossack you must call Apple support, no other way. I called apple support from Croatia cause I'm croatian and begged they connect me to USA apple and after few calls I got helped. My name in database was written as MATKQ, and not MATKO, so when I wanted to generate expo app I was getting errors of Apple developer not present.
